addon/addon/components/animation-context/index.ts

```diff
@@ -18,17 +18,20 @@ const { VOLATILE_TAG, consumeTag } =
   Ember.__loader.require('@glimmer/validator');
 
 interface AnimationContextArgs {
-  Args: {
-    id?: string;
-    use: ((changeset: Changeset) => AnimationDefinition) | undefined;
-  };
+  id?: string;
+  use: ((changeset: Changeset) => AnimationDefinition) | undefined;
+}
+
+interface Signature {
+  Element: HTMLDivElement;
+  Args: AnimationContextArgs;
   Blocks: {
-    default: [];
+    default: [AnimationContextComponent];
```

Comment:
No idea why we still yield this, but I don't think we have any use case for it. We can remove it (either in this PR or immediately after we merge this I suppose).